### PR TITLE
Add SDLC

### PR DESCRIPTION
**Pull request by ChrisScottThomas**

### CODE REVIEW

File: docs/Engineering/Procedures/pr.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 4
+sidebar_position: 2
```
Comment (connoravo) [inline docs/Engineering/Procedures/pr.md:2]: Would need to change position if SDLC is to take 2nd

File: docs/Engineering/Procedures/sdlc.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+---
+
+# The Burendo Software Development Life Cycle (SDLC)
+
+![SDLC](images/sdlc.png)
+
+
+## What is role of the SDLC?
+
+The SDLC presents a consistent and standardised, yet flexible framework that helps users define activities and deliverables(work).  Using this framework we can build a picture, that can be understood at all levels, of how work can be taken from inception to implementation and beyond.  Once in place and its concepts are used by all, it creates higher velocity of output, increased trust between all parties involved and more creative freedom within the bounds set by the framework.
```
Comment (connoravo) [inline docs/Engineering/Procedures/sdlc.md:12]:
```suggestion
The SDLC presents a consistent and standardised, yet flexible framework that helps users define activities and deliverables (work).  Using this framework we can build a picture, that can be understood at all levels, of how work can be taken from inception to implementation and beyond.  Once in place and its concepts are used by all, it creates higher velocity of output, increased trust between all parties involved and more creative freedom within the bounds set by the framework.
```

File: docs/Engineering/Procedures/sdlc.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+- How do we formulate improvements?
+- What security concerns do we need to understand?
+
+## The Six Stages
+
+The following six stages make up the cyclical process most people are familiar with, building off the foundational concepts covered above.  We are able to confidently identify _what_ we want to do within each stage, without getting into the how.  The how, as explained above in the [Core concept](#core), will be heavily influenced by many factors and will inevitably look quite different between implementations.
+
+### Inception
+
+[The beninging](https://www.youtube.com/watch?v=vacJSHN4ZmY).  The traditional front door.  Generally there are two main drivers at this stage, work driven by business value and work driven by improvements on previous work.
```
[the remark this answers is not in the document]
Comment (ChrisScottThomas) [reply docs/Engineering/Procedures/sdlc.md:67]: ?

File: docs/Engineering/Procedures/sdlc.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+---
+
+# The Burendo Software Development Life Cycle (SDLC)
+
+![SDLC](images/sdlc.png)
+
+
+## What is role of the SDLC?
+
+The SDLC presents a consistent and standardised, yet flexible framework that helps users define activities and deliverables (work).  Using this framework we can build a picture, that can be understood at all levels, of how work can be taken from inception to implementation and beyond.  Once in place and its concepts are used by all, it creates higher velocity of output, increased trust between all parties involved and more creative freedom within the bounds set by the framework.
```
Comment (brianbayes) [inline docs/Engineering/Procedures/sdlc.md:12]: Think _presents_ should be _represents_

File: docs/Engineering/Procedures/sdlc.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+## The four D's
```
Comment (brianbayes) [inline docs/Engineering/Procedures/sdlc.md:14]: I'm not sure this needs an apostrophe, i.e. should it be "The four Ds"? I think four should be title-cased, i.e. Four as per the other ## Headers

File: docs/Engineering/Procedures/sdlc.md (hunk 4 above)
Comment (ChrisScottThomas) [reply docs/Engineering/Procedures/sdlc.md]: Fixed

File: docs/Engineering/Procedures/sdlc.md (hunk 5 above)
Comment (ChrisScottThomas) [reply docs/Engineering/Procedures/sdlc.md]: Fixed

**Review by steveburtonBUR — APPROVED** [review]

**Pull request merged**